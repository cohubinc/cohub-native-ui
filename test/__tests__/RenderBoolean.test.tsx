import React from "react";
import { mount } from "enzyme";

import { RenderBoolean } from "dist";

it("matches snapshot", async () => {
  const wrapper = mount(<RenderBoolean value={true} />);

  expect(wrapper).toMatchSnapshot();
});
