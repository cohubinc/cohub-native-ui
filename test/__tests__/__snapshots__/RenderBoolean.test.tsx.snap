// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<RenderBoolean
  value={true}
>
  <span>
    Yes
  </span>
</RenderBoolean>
`;
